catboost/python-package/ut/medium/test.py

```diff
@@ -6875,3 +6877,55 @@ def test_langevin_with_empty_leafs():
     for value, weight in zip(model.get_leaf_values(), model.get_leaf_weights()):
         if weight == 0:
             assert value == 0
+
+
+def test_to_classifier():
+    train_pool = Pool(TRAIN_FILE, column_description=CD_FILE)
+    test_pool = Pool(TEST_FILE, column_description=CD_FILE)
+
+    models = []
+    for learning_rate in [.03, .05]:
+        models.append(CatBoostClassifier(iterations=2, learning_rate=learning_rate))
+        models[-1].fit(train_pool, eval_set=test_pool)
+
+    merged_model = sum_models(models)
+    prediction = merged_model.predict(test_pool, prediction_type='Probability')
+
+    merged_model = to_classifier(merged_model)
+
+    assert isinstance(merged_model, CatBoostClassifier)
+    assert _check_data(merged_model.predict(test_pool, prediction_type='Probability'), prediction)
```

Comment:
`CatBoostClassifier` has special method `predict_proba`, it will be better to use it here.